journal/src/main/scala/com/evolutiongaming/kafka/journal/JsValueCodec.scala

```diff
@@ -0,0 +1,70 @@
+package com.evolutiongaming.kafka.journal
+
+import cats.Applicative
+import cats.implicits._
+import com.evolutiongaming.catshelper.FromTry
+import com.evolutiongaming.jsonitertool.PlayJsonJsoniter
+import play.api.libs.json.{JsValue, Json}
+import scodec.bits.ByteVector
+
+import scala.util.Try
+
+case class JsValueCodec[F[_]](encode: JsValueCodec.Encode[F], decode: JsValueCodec.Decode[F])
+
+object JsValueCodec {
+
+  def apply[F[_]](implicit ev: JsValueCodec[F]): JsValueCodec[F] = ev
+
+  case class Encode[F[_]](toBytes: ToBytes[F, JsValue])
+
+  object Encode {
+    implicit def fromCodec[F[_]](implicit codec: JsValueCodec[F]): Encode[F] = codec.encode
+  }
+
+  case class Decode[F[_]](fromBytes: FromBytes[F, JsValue])
+
+  object Decode {
+    implicit def fromCodec[F[_]](implicit codec: JsValueCodec[F]): Decode[F] = codec.decode
+  }
+
+  def playJson[F[_] : Applicative : FromTry]: JsValueCodec[F] = JsValueCodec(
+
+    encode = Encode { value =>
+      ByteVector(Json.toBytes(value)).pure[F]
+    },
+
+    decode = Decode { bytes =>
+      lift {
+        Try(Json.parse(bytes.toArray))
+      }
+    }
+  )
+
+  def jsoniter[F[_] : Applicative : FromTry]: JsValueCodec[F] = JsValueCodec(
```

Comment:
@plokhotnyuk thanks for your work on jsoniter, just to let you know: we are going to use jsoniter in one of the most loaded part of our applications :)

Reply:
+1